packages/csv-stringify/lib/api/index.js

```diff
@@ -158,6 +158,9 @@ const stringifier = function(options, state, info){
             }
           });
           quotedMatch = quotedMatch && quotedMatch.length > 0;
+          if (escape_formulas && ['=', '+', '-', '@', '\t', '\r'].includes(value[0])) {
```

Comment:
Shouldn't this happen in the `__onField` function ?

Reply:
I think __onField is only in csv-parse. not csv-stringify